### PR TITLE
Fixes disposals, transit Tcomms, and a door

### PR DESCRIPTION
**Pull request by nik707**
## About The Pull Request

Adds tcomms to midpoint, fixes disposals, and removes the gaming den hallway door ebcause it doesn'T WORK AND IC A N'T FIGURE OUT AHNYEWAHATGEWG

## Why It's Good For The Game

idk comms are nice

## Changelog
:cl:

fix: fixed a few things

/:cl:

### CODE REVIEW

File: maps/tether/tether-03-surface3.dmm [hunk 1]
```diff
@@ -23598,12 +23601,6 @@
 	},
 /turf/simulated/floor/plating,
 /area/tether/surfacebase/public_garden_three)
-"Ym" = (
-/obj/machinery/door/airlock/glass{
-	name = "The Game Den"
-	},
-/turf/simulated/floor/tiled/steel_grid,
-/area/tether/surfacebase/outside/outside3)
```
Comment (TheFurryFeline) [inline maps/tether/tether-03-surface3.dmm]: `/area/tether/surfacebase/outside/outside3`
This is why it wasn't working. Change it to be either the bar's area or the hallway's area.

Comment (TheFurryFeline) [reply maps/tether/tether-03-surface3.dmm]: Trust me, I'm a mapper. :3

**Comment by buffyuwu** [comment]
areas distribute power! listen to the good kitty <3

**Comment by KillianKirilenko** [comment]
I could've *sworn* the areas on the doors looked fine last time I looked into this... but sure enough, they're not.

image

Those two window tiles will need their areas fixing too, or else those emergency shutters might not work properly. Which will be bad if their associated windows are busted open.

**Comment by TheFurryFeline** [comment]
> I could've _sworn_ the areas on the doors looked fine last time I looked into this... but sure enough, they're not.
> 
> image
> 
> Those two window tiles will need their areas fixing too, or else those emergency shutters might not work properly. Which will be bad if their associated windows are busted open.

Oh, aye, that sort of thing would be a priority to fix. I'll see if Virgo's tether for the bar's windows have the same issues.

**Comment by nik707** [comment]
I literally looked at areas in the mapper and it didn't show that, what the fuck

**Comment by KillianKirilenko** [comment]
Probably the mapmerge scripts freaking out and trying to fix stuff that isn't actually broken. I think it tries to 'fix' things that seem to be duplicated and sometimes has hiccups like these.

**Pull request closed without merging**